samples/package.json

```diff
@@ -14,7 +14,7 @@
     "test": "repo-tools test run --cmd ava -- -T 5m --verbose system-test/*.test.js"
   },
   "dependencies": {
-    "@google-cloud/video-intelligence": "1.1.0",
+    "@google-cloud/video-intelligence": "^1.1.0",
```

Comment:
Why is that? Previously we always pinned the version of the main package.

Reply:
This was auto-added. I figured the script that added it automatically had a legit reason. I can remove it if it should not be there.

Reply:
Yes, please pin it `"1.1.0"`, just to match all other libraries.

Reply:
Should I remove this @alexander-fenster or let it be as it is?

Reply:
Yes, let's leave `"@google-cloud/video-intelligence": "1.1.0"` just as we do for other packages.